setup.py

```diff
@@ -119,7 +130,7 @@ def is_platform_mac():
         "h5py",
         "setuptools_scm",
     ],
-    "tests_require": ["pytest", "pytest-cases"],
+    "tests_require": ["pytest", "pytest-xdist", "pytest-cases>=1.12.1"],
```

Comment:
I think we should actually remove this line entirely. Calling tests from setup.py is deprecated: https://github.com/pypa/setuptools/issues/1684